### PR TITLE
Release/0.5.0

### PR DESCRIPTION
**Pull request by jbeemster**

### CODE REVIEW

**Review by Andy-Hay — APPROVED** [review]
LGTM

File: main.tf [hunk 1]
```diff
@@ -407,24 +407,34 @@ locals {
   })
 }
 
-resource "aws_launch_configuration" "lc" {
+resource "aws_launch_template" "lt" {
```
Comment (Andy-Hay) [inline main.tf]: One suggestion would be to add `tags = local.tags` in here so that the launch template resource has tagging added.

**Comment by angelsk** [comment]
I'm sure you don't need me to tell you that there's still some hardcoded links to your documents folder for the telemetry... :)

**Review by Andy-Hay — APPROVED** [review]
LGTM

**Pull request merged**